docs/e2e_tests.md

```diff
@@ -18,23 +24,53 @@ The flow is as follows:
 10. Verify if BTP Operator has the `Ready` status equal to `True`.
 11. Create a Service Instance and Service Binding with either real or dummy credentials. 
 12. When real credentials are used, verify if the Service Instance and Service Binding have the `Ready` status `True`. If dummy credentials are used, verify if the `Ready` status for both of them is`NotProvisioned`.
-13. Uninstall BTP Operator.
-14. Uninstall BTP Manager. 
+13. Try to uninstall BTP Operator without force delete label.
```

Comment:
Why are you saying `Try to uninstall...` and not `Uninstall..` 
In case of `Try to...`, I'd expect further instructions such as `If you succeed ...` and `If you don't succeed...` WDYT?